### ISSUE TITLE
is there any inspect tool can be used?

### ISSUE THREAD

**Issue by meskmanz** [appium/appium-mac2-driver#45]
Hi guys,
I've been using appium-for-mac + Java + selenium and accessibility inspector as an inspect tool. Now I'm thinking on migrating to appium-mac2-driver but I cannot find any inspect tool that can be used.
Could you please advise some inspect tool for appium-mac2-driver ?

**Comment by mykola-mokhnach** [comment]
I think right now there is no reliable tool for this purpose. I assume that https://github.com/mykola-mokhnach/Appium-iOS-Inspector with slight modifications could do the trick

**Comment by sramssh** [comment]
I couldn't find a good tool.  I print the pagesource and then identify the locators.

**Comment by mykola-mokhnach** [comment]
There is a a couple of third-party tools which allow to inspect macOS accessibility: https://forum.keyboardmaestro.com/t/os-x-accessibility-inspector-uielementinspector-tool-for-ui-scripting/3443
